Yauheni_Mandrik_HTML_CSS/index.html

```diff
@@ -0,0 +1,392 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&family=Yellowtail&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/styles.css">
+    <link rel="stylesheet" href="./styles/media.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="site">
+        <header class="header  content">
+            
+
+                <div class="header_logo">
+                    <img src="./img/logo.png" alt="logo">
+                </div>
+    
+                <nav class="header_nav">
+                    <label class="burger_menu" for="checkbox">
+                        <span></span>
+                        <span></span>
+                        <span></span>
+                    </label>
+                    <input type="checkbox"  id="checkbox">
+                    <ul >
+                        <li><a href="#">Home</a></li>
+                        <li><a href="#">About Us</a></li>
+                        <li><a href="#">Shop</a></li>
+                        <li><a href="#">Pages</a></li>
+                        <li><a href="#">Contact Us</a></li>
+                        <li><button class="header_basket"><img src="./img/basket.png" alt=""><span>Cart</span><span>0</span></button></li>
+
+                        
+                    </ul>
+                    
+                </nav>
+    
+                
+                    
+                
+
+           
+        </header>
+
+        <article class="promo">
```

Comment:
https://www.w3schools.com/tags/tag_article.asp
`the only way <article><h><p></>`
replace section pls